lib/recurly/token.js

```diff
@@ -145,5 +145,15 @@ function validate (input) {
     errors.push('last_name');
   }
 
+  if (input.cvv && !this.validate.cvv(input.cvv)) {
+    errors.push('cvv');
+  }
```

Comment:
We actually don't require `cvv` in all cases, so this should definitely be optional.